web_src/js/globals.d.ts

```diff
@@ -16,6 +16,7 @@ declare module '*.vue' {
   // To actually lint .vue files, `vue-tsc` is used because `tsc` can not parse them.
   export function initDashboardRepoList(): void;
   export function initRepositoryActionView(): void;
+  export type Item = any;
```

Comment:
But why, which module needs it, why it appears in the global scope.

Reply:
Why `Item` is exported, which module is it exported from?

Such a general name should never be designed to be exported.

Reply:
moved and renamed it.